matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/DefaultSession.kt

```diff
@@ -174,8 +174,8 @@ internal class DefaultSession @Inject constructor(
             lifecycleObservers.forEach {
                 it.onSessionStarted(this)
             }
-            sessionListeners.dispatch { _, listener ->
-                listener.onSessionStarted(this)
```

Comment:
as we were already avoiding the `session` parameter in the callback it felt like the `SessionListeners` didn't actually need to hold the session itself

this also means that the `close` flow can safely use the session instance without worrying about it dissapearing from the `SessionManager` which would cause `onSessionStopped` to not be called